Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WindowsRuntimeCollection.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    public sealed class WindowsRuntimeCollection<T1, T2> : IList<T1>
```

Comment:
Looks like our styler isn't catching things on the build?

We should have a doc comment on public classes/properties.

Reply:
Thanks, this should be internal